provision-dns-server.sh

```diff
@@ -92,7 +100,12 @@ pdnsutil load-zone $k0s_domain <(echo "
     1m         ; minimum (the slave stores negative results for this long)
 )
                 IN      NS      a.ns
-")
+EOF
+
+# pdnsutil doesn't support loading from non-regular files.
```

Comment:
that's interesting. you mean that the debian-12 powerdns version is way behind ubuntu one?